src/core/utils.ts

```diff
@@ -28,3 +29,13 @@ export function mapToSearchActions(values: string[], actions: typeof FluxCapacit
 }
 
 export const rayify = <T>(values: T | T[]): T[] => Array.isArray(values) ? values : [values];
+
+export const arrayToDotNotation = (input: string, throwError = false): string => {
```

Comment:
why do we have the `throwError` parameter?

Reply:
Sorry, that was left in from when I thought it would be a good idea to have that option. But since we're going to throw errors I've removed that boolean.